events/wmt.md

```diff
@@ -0,0 +1,103 @@
+
+During WMT's first years, the event held a series of workshops within the Empirical Methods in Natural Language Processing (EMNLP) conference organised by the Association for Computational Linguistics (ACL). In 2016, with the uprise of [neural machine translation](approaches/neural-machine-translation.md), WMT became a conference on its own, maintaining its name.
+
+> The conference aims to bring together academic scientists, researchers and industry representatives to exchange and share their experiences and research results. WMT plays a key role for the entire industry of computational linguistics and machine translation.
+
+## Shared tasks
+
+WMT hosts shared tasks competitions prior to the conference. During the shared tasks, researchers tackle specific challenges from a list of themes. Researchers then compare their techniques against the techniques of other researchers in the field.
+
+During the main conference, researchers present the results of the shared tasks and winners are announced.
+
+WMT started in 2006 with a *translation* task. In the following editions, WMT included themes on all aspects of machine translation, corpus preparation, training, and evaluation.
+
+### 2022 shared tasks
+
+- General machine translation task (former News task)
+- Biomedical translation task
+- Automatic post-editing task
+- Metrics task
+- Quality estimation task
+- Machine translation using terminologies task
+- Chat translation task
+- Sign language translation task
+- Robustness translation task
+- Triangual machine translation task
+- Large-scale multilingual machine translation task
+
+The results published from WMT shared tasks represent benchmarks for the technology.
+
+
+## Participants
+
+Universities and big tech companies consistently participate in the conference and are represented in the organising committee.
+
+### Participating universities
+
+*Incomplete list*
+
+- University of Edinburgh
+- Johns Hopkins University
+- Charles University
+- University of Amsterdam
+- University of Dublin
+- Fondazione Bruno Kessler
+- University of Amsterdam
+
+### Participating companies
+
+*Incomplete list*
+
+- IBM
+- Google
+- Amazon
+- Microsoft
+- Apple
+- Unbabel
+
+### Long-term organisers
+
+- [Philipp Koehn](community/people/philipp-koehn.md) (University of Edinburgh / Johns Hopkins University)
+- Barry Haddow (University of Edinburgh)
+- Loïc Barrault (University of Sheffield)
+- Ondřej Bojar (Charles University)
+- Marco Turchi (Fondazione Bruno Kessler)
+- Martin Popel (Charles University)
+- Matt Post (Johns Hopkins University)
+- Rajen Chatterjee (Apple)
+- Christof Monz (University of Amsterdam)
+- Matteo Negri (Fondazione Bruno Kessler)
+- Marco Turchi (Fondazione Bruno Kessler)
+- Matthias Huck (University of Edinburgh)
+- Christian Federmann (Microsoft)
+- Christof Monz (University of Amsterdam)
+
+## History
```

Comment:
I wouldn't call this "History", but not sure what to call it.

Could just put `---` to make a separator.